session1/excercises/generics.ts

```diff
@@ -0,0 +1,11 @@
+class mobile{
+    constructor(public modelName: string,public price: number) {        
+    }
+}
+
+function samsung<T extends mobile>(c: T) {
+    console.log(c.modelName, c.price);
+}
+
+let galaxyNote = new mobile("Samasung Galaxy Note 10", 70000);
+samsung(galaxyNote);
```

Comment:
Why are we printing the entire object here?
We should rather be printing its member variables as mentioned in the problem description too.